ambry-router/src/test/java/com.github.ambry.router/DefaultKeyManagementServiceTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.router;
+
+import com.github.ambry.account.Account;
+import com.github.ambry.account.Container;
+import com.github.ambry.config.KMSConfig;
+import com.github.ambry.config.VerifiableProperties;
+import com.github.ambry.utils.TestUtils;
+import com.github.ambry.utils.Utils;
+import java.util.Properties;
+import java.util.Random;
+import javax.crypto.spec.SecretKeySpec;
+import org.bouncycastle.util.encoders.Hex;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static com.github.ambry.router.CryptoUtils.*;
+
+
+/**
+ * Tests {@link DefaultKeyManagementService} and {@link DefaultKeyManagementServiceFactory}
+ */
+public class DefaultKeyManagementServiceTest {
+
+  private static final Random random = TestUtils.RANDOM;
```

Comment:
Use `TestUtils.RANDOM` directly. What is the intent behind creating a local variable?